yarn/src/main/scala/org/apache/spark/deploy/yarn/Client.scala

```diff
@@ -836,8 +866,8 @@ private[spark] class Client(
     val amArgs =
       Seq(amClass) ++ userClass ++ userJar ++ primaryPyFile ++ primaryRFile ++
         userArgs ++ Seq(
-          "--executor-memory", args.executorMemory.toString + "m",
-          "--executor-cores", args.executorCores.toString,
+          "--executor-memory", executorMemory.toString + "m",
```

Comment:
@vanzin , so you mean we also need to simplify the command line argument for AM?